packages/tasks/src/tasks/mask-generation/about.md

```diff
@@ -0,0 +1,43 @@
+## Use Cases
+
+### Filtering an Image
+
+The mask might serve as an initial filter to eliminate irrelevant information. For instance, when monitoring vegetation in satellite imaging, a mask can be created to identify green spots, highlighting the relevant region of the image.
+
+### Pre-processing
+
+Generating masks that can to separate background and foreground (or other interesting properties) in an image can be done to facilitate learning, especially in semi- or unsupervised learning.
+
+### Human-in-the-loop
```

Comment:
Can you elaborate the use case here? i.e. The name of the use case felt a little ambiguous